lib/kitchen/directions/bake_inline_lists.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bakes inline lists with the desired list separator
+    # Does not separate the last list item
+    #
+    module BakeInlineLists
+      def self.v1(book:)
+        list_separator = '; '
+        separator_class = '-os-inline-list-separator'
```

Comment:
since these strings don't change, make them constants...

```ruby
module BakeInlineLists
  LIST_SEPARATOR = '; '
  SEPARATOR_CLASS = '-os-inline-list-separator'

  def self.v1(book:)
```